apps/root/src/hooks/useOpenConnectModal.ts

```diff
@@ -11,71 +11,74 @@ const useOpenConnectModal = () => {
+        }
 
-      if (textsContainer) {
-        textsContainer.className = `${textsContainer.className} rainBowKitTextsContainer ${mode}`;
-        let loginText = defineMessage({
-          defaultMessage: 'Log in with your wallet',
-          description: 'RainbowLogInWithYourWallet',
-        });
-        let loginSubText = defineMessage({
-          defaultMessage:
-            "Log in with the first wallet you registered to access your account, including any other wallets and contacts you've added.",
-          description: 'RainbowLogInWithYourWalletSubText',
-        });
-        let loginDisclaimer = defineMessage({
-          defaultMessage: 'Logging in with a different wallet creates a new account without your existing data.',
-          description: 'RainbowLogInDisclaimer',
-        });
-        let loginWalletTitle = defineMessage({
-          defaultMessage: 'Select your Wallet to Log In.',
-          description: 'RainbowWalletTitle',
-        });
+        const textsContainer = document.querySelector(
+          'div.iekbcc0.ju367v4.ju367vf0.ju367va.ju367v15.ju367v2m.ju367v2s'
+        );
 
-        if (user?.status === UserStatus.loggedIn) {
-          loginText = defineMessage({
-            defaultMessage: 'Link a new wallet',
-            description: 'RainbowLinkANewWallet',
+        if (textsContainer) {
+          textsContainer.className = `${textsContainer.className} rainBowKitTextsContainer ${mode}`;
+          let loginText = defineMessage({
+            defaultMessage: 'Log in with your wallet',
+            description: 'RainbowLogInWithYourWallet',
           });
-          loginSubText = defineMessage({
-            defaultMessage: 'Connect another wallet to your already logged-in wallet',
-            description: 'RainbowLinkANewWalletSubText',
+          let loginSubText = defineMessage({
+            defaultMessage:
+              "Log in with the first wallet you registered to access your account, including any other wallets and contacts you've added.",
+            description: 'RainbowLogInWithYourWalletSubText',
           });
-          loginDisclaimer = defineMessage({
-            defaultMessage: 'Linking wallets allows you to track more than one!',
-            description: 'RainbowLinkWalletDisclaimer',
+          let loginDisclaimer = defineMessage({
+            defaultMessage: 'Logging in with a different wallet creates a new account without your existing data.',
+            description: 'RainbowLogInDisclaimer',
           });
-          loginWalletTitle = defineMessage({
-            defaultMessage: 'Select your Wallet to link it.',
+          let loginWalletTitle = defineMessage({
+            defaultMessage: 'Select your Wallet to Log In.',
             description: 'RainbowWalletTitle',
           });
-        }
 
-        const titleContainer = document.querySelector('[data-testid="rk-connect-header-label"]');
+          if (user?.status === UserStatus.loggedIn && !isReconnecting) {
```

Comment:
Are you formatting documents manually?

Reply:
WTF

Reply:
Oh this happened because I added a parameter to the function, so everything is idented